user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -27,3 +29,81 @@ Please note the following regarding suggestion events:
+
+   - If you approve of the anomaly and want to receive similar ones in the future, click the thumbs-up icon in the feedback column. Alternatively, you can right-click the suggestion event/alarm and select *Like*.
+
+   - If you disapprove of the anomaly and do not wish to be notified of similar events in the future, click the thumbs-down icon. Alternatively, you can right-click the suggestion event/alarm and select *Dislike*.
+
+   Once selected, the chosen icon will be pinned in the column as a black icon. If you change your mind, you can click the other option, and only the final feedback will be registered<!--RN 39082-->.
+
+   > [!NOTE]
+   >
+   > - The ![Thumbs-up](~/user-guide/images/Thumbs_Up.png) or ![Thumbs-down](~/user-guide/images/Thumbs_Down.png) icons will only appear for alarms or suggestion events generated by the behavioral anomaly detection feature.
+   > - You can only see the feedback icons you have clicked yourself. Feedback from other users is not visible.
+
+The feedback you provide will be taken into account in the future.
+
+Feedback on active alarms is saved in the user settings and will re-appear after closing and reopening Cube. Feedback on history alarms will disappear after restarting Cube, but it will still be used by SLAnalytics to improve the detection of future anomalies.
+
+### Follow-up actions after providing feedback
+
+After providing feedback on a suggestion event or alarm, a light bulb icon may appear next to the ![Feedback](~/user-guide/images/Feedback_Column.png) column. Clicking this icon will reveal a follow-up action based on your feedback<!--RN 39809 + 39640 + 39480-->. Alternatively, you can right-click the suggestion event or alarm and select the follow-up action from the context menu.
+
+| Action | Description |
+|--|--|
+| Clear event | Clears the suggestion event immediately. No confirmation box will appear. |
+| Improve alarm template | Opens a pop-up window suggesting changes to the alarm template, based on previous feedback for the parameter. At the top of the window, you can see the element and parameter name, along with a link to relevant documentation<!--RN 39616-->. At the bottom, it shows which elements using the same alarm template will be affected by the suggested template changes<!--RN 39729-->. A clock button allows you to view the current anomaly template configuration<!--RN 39640-->. |
+| Create alarm template | Opens a card to create a new alarm template from scratch. |
+
+![Suggested actions](~/user-guide/images/Suggested_Actions.png)<br>*Alarm Console in DataMiner 10.4.11*
+
+If you provide feedback on multiple suggestion events for the same parameter, an action will only be suggested for the last one you gave feedback on<!--RN 39640-->.
+
+> [!NOTE]
+>
+> - The actions listed above will only appear if you have the required permissions to perform them. For example, the system will not suggest updating the alarm template if you are not allowed to do so<!--RN 39480-->.
```

Comment:
Maybe we should mention the exact permissions here? That way we prevent questions about this.

Reply:
Added permissions in [commit 0464372](https://github.com/SkylineCommunications/dataminer-docs/pull/3761/commits/0464372e917723612ee45bc22bab1eba770d33e1)

![image](https://github.com/user-attachments/assets/daf61cef-c376-4312-89b4-faf8c12a2d05)